code/modules/mob/dead/observer/observer.dm

```diff
@@ -205,9 +205,14 @@ var/list/image/ghost_images_simple = list() //this is a list of all ghost images
 	b_val += (255 - b_val) * 0.4
 	if(b_val > 255)
 		b_val = 255
-
+	
 	return num2hex(r_val, 2) + num2hex(g_val, 2) + num2hex(b_val, 2)
 
+/mob/proc/whiten_color() 
+	var r_val = 246
+	var g_val = 246
+	var b_val = 246
```

Comment:
The formatting works but it's not the standard we're using for tg.

@GunHog You don't need to test anything to see that this code does nothing, it's really obvious.